src/main/npl-1.0/objects/iou/iou.npl

```diff
@@ -38,6 +39,11 @@ protocol[issuer, payee] Iou(var forAmount: Number) {
         become forgiven;
     };
 
+    @api
+    permission[payee] confirmRepaid() | repaid {
+        become confirmed_repaid;
+    };
```

Comment:
These are behavioural changes that weren't specified in the ticket... Are they necessary?

Once (if) we've made changes to the obligation semantics, we'll revisit these examples and presumably use obligations correctly. But for now I'd leave out any changes unless there is a good reason to introduce them. This diff would be a perfect example of some business logic to add in a demo, which can then be deployed in order to show the new API being generated.

Reply:
The content changes to the IoU file were uninted, thanks for noticing, should have checked the diff. Fixed